code/modules/cargo/exports/weapons.dm

```diff
@@ -174,12 +174,12 @@
 /////////////////
 
 /datum/export/weapon/wtammo
-	cost = 15
+	cost = 500
```

Comment:
I think someone somewhere might have made WT550 ammo a techweb item for security, so this would be an insanely profitable farming method. You need someone to click, but every click is 500 credits.

Reply:
Fair, I'll turn it down to 100 as to not make it that useless.